ios/FluentUI/Pill Button Bar/PillButton.swift

```diff
@@ -182,8 +203,17 @@ open class PillButton: UIButton {
         if let window = window {
             if isSelected {
                 if isEnabled {
-                    backgroundColor = style.selectedBackgroundColor(for: window)
-                    setTitleColor(style.selectedTitleColor(for: window), for: .normal)
+                    if let customSelectedBackgroundColor = customSelectedBackgroundColor {
```

Comment:
since you're making these changes, could you also change lines 222 to 226 to follow the same pattern?

`backgroundColor = customBackgroundColor ?? style.backgroundColor(for: window)`